### PR TITLE
Added note regarding how to disable auto scaling

### PR DESCRIPTION
**Pull request by Templum**
Signed-off-by: Simon Pelczer <templum.dev@gmail.com>

### CODE REVIEW

File: docs/architecture/autoscaling.md [hunk 1]
```diff
@@ -16,13 +16,12 @@ The AlertManager rules ([alert.rules](https://github.com/openfaas/faas/blob/mast
...
 The minimum (initial) and maximum replica count can be set at deployment time by adding a label to the function.
 
-* `com.openfaas.scale.min`
+* `com.openfaas.scale.min` By default this is set to `1`
 
-By default this is set to `1`
+* `com.openfaas.scale.max` The current default value is `20` for 20 replicas
 
-* `com.openfaas.scale.max`
+> Note: Setting `com.openfaas.scale.max=1` allows to disable the auto-scaling functionality of openfaas.
```
[the remark this answers is not in the document]
Comment (Templum) [reply docs/architecture/autoscaling.md]: The mention was based on what was discussed [here](https://github.com/openfaas/faas/issues/541). However, to be correct it is the same as Min Replicas. I will make adjustments to highlight this.

File: docs/architecture/autoscaling.md [hunk 2]
```diff
@@ -16,13 +16,12 @@ The AlertManager rules ([alert.rules](https://github.com/openfaas/faas/blob/mast
 
 The minimum (initial) and maximum replica count can be set at deployment time by adding a label to the function.
 
-* `com.openfaas.scale.min`
+* `com.openfaas.scale.min` By default this is set to `1`
```
[the remark this answers is not in the document]
Comment (Templum) [reply docs/architecture/autoscaling.md]: Well, that might be an aesthetic question. With the comment, In my opinion, it looked better if the information was on the same line. But I can change that

File: docs/architecture/autoscaling.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Templum) [reply docs/architecture/autoscaling.md]: Clarified the note in f1c546c

**Comment by alexellis** [comment]
Thank you Simon

**Pull request merged**